nebullvm/optimizers/multi_compiler.py

```diff
@@ -40,7 +42,10 @@ def _tvm_is_available() -> bool:
 
 
 def select_compilers_from_hardware():
-    compilers = [ModelCompiler.ONNX_RUNTIME]
+    compilers = [
+        ModelCompiler.ONNX_RUNTIME,
+        ModelCompiler.ONNX_MLIR_RUNTIME,
```

Comment:
Since you need to install both LLVM and ONNX-MLIR from source to install ONNX-MLIR, I would avoid adding it to the compiler list by default (because it takes a long time to install). I would prefer to treat it similarly to TVM, i.e. checking its installation with a `try except` code.